zt_frontend/src/components/Cell.vue

```diff
@@ -243,7 +243,7 @@
       </div>
       <div
         :class="['outcome', { 'outcome--dev': isDevMode }]"
-        v-if="!(isDevMode && !isAppRoute && cellType === 'text')"
+        v-if="!(isDevMode && !isAppRoute && cellType === 'text') && cellHasOutput"
```

Comment:
This should be removed, this is only used in app mode and with this the output is not rendered in app mode